CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # AmpliPi Software Releases
 
+## 0.4.5
+* Web App
+  * Add error handling on browser page for instances where the selected stream isn't browsable
+
+## 0.4.4
+* Web App
+  * Ensure that media device audio streams stop outputting audio when disconnected
+  * Limit max height of group creation dropdown
+  * Improve how marquee components look when not scrolling
```

Comment:
due to the ex-post-facto nature of 0.4.4, [the changelog](https://github.com/micro-nova/AmpliPi/releases/tag/0.4.4) isn't in our current git history on main